src/main/java/org/spongepowered/mod/mixin/core/common/MixinSpongeImplHooks.java

```diff
@@ -286,4 +293,23 @@ public static void onTileEntityError(TileEntity tileEntity, CrashReport crashRep
     public static void blockExploded(Block block, World world, BlockPos blockpos, Explosion explosion) {
         block.onBlockExploded(world, blockpos, explosion);
     }
+
+    // Crafting
+
+    @Overwrite
```

Comment:
Please add a Javadoc comment with `@author` and `@reason` for both overwrites (Mixin will print compile warnings if they're missing)